eng/versioning/version_client.txt

```diff
@@ -143,8 +143,7 @@ com.azure:azure-security-keyvault-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template;1.1.1234;1.2.2-beta.1
 com.azure:azure-sdk-template-two;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template-three;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-spring-data-cosmos;3.26.0;3.27.0-beta.1
-com.azure:azure-spring-data-cosmos-test;3.0.0-beta.1;3.0.0-beta.1
+com.azure:azure-spring-data-cosmos;6.0.0-beta.1;6.0.0-beta.1
```

Comment:
current versions of azure-spring-data-cosmos is v3. Why do we have the version as 6.0.0-beta.1 ?
What is the plan for releasing multiple major versions of azure-spring-data-cosmos?